airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -44,6 +44,8 @@ tests:
     - config_path: "secrets/connected_account_config.json"
       configured_catalog_path: "integration_tests/connected_account_configured_catalog.json"
       ignored_fields:
-        "invoices":
-          - invoice_pdf
-          - hosted_invoice_url
+        invoices:
+          - name: invoice_pdf
+            bypass_reason: "fast changing data"
+          - name: hosted_invoice_url
+            bypass_reason: "fast changing data"
```

Comment:
```suggestion
        invoices:
          - name: invoice_pdf
            bypass_reason: "URL changes upon each request for privacy/security"
          - name: hosted_invoice_url
            bypass_reason: "URL changes upon each request for privacy/security"
```

Reply:
For streams that really have changing data, we need to add better reasons to explain why